.github/workflows/hma-ci.yaml

```diff
@@ -77,5 +77,5 @@ jobs:
       - name: Test with pytest on docker image
         run: |
           docker run --network="host" local sh -c "python -m pip install --upgrade pip ;
-            pip install -e '.[test]' ;
-            py.test"
+            pip install pytest ;
```

Comment:
Hmm, less of a fan of this change, the reason to use pip install is because if you need to add additional testing requirements, they can listed in the setup.